packages/x-scheduler/src/joy/event-calendar/EventCalendar.types.ts

```diff
@@ -0,0 +1,12 @@
+import { CalendarEvent } from '../models/events';
+
+export interface EventCalendarProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * The events to render in the calendar.
+   */
+  events: CalendarEvent[];
+  /**
+   * Callback fired when some event of the calendar change.
+   */
+  onEventsChange?: (value: CalendarEvent[], event: Event) => void;
```

Comment:
I followed blindly the signature of the Base UI callbacks to keep things consistent :laughing:

Reply:
When would this callback be called? 🤔 
Is it a global callback that you expect we'll call whenever we add, delete or modify an event? 🤔 
If that is the case, I suspect we'll need a context object. 🙈 

P.S. Where is this approach used in base-ui?

Reply:
But yes, for me it's the global callback that would be called whenever you add / delete / modify an event (since by default you are controlling the events).

Reply:
On the new version of the pickers we will clearly follow there callback structure